templates/themes/1/home.amp.html

```diff
@@ -0,0 +1,109 @@
+<!---
+Copyright 2017 The AMPStart Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!doctype html>
+{{#blog-photogallery}}
```

Comment:
Since this is an article template, I would rename it to article.amp.html and update `blog-photogallery` to `article` too.